ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -32,6 +32,7 @@ CLUSTER_SECTOR=${CLUSTER_SECTOR:-}
 ADDITIONAL_SECURITY_GROUP=${ADDITIONAL_SECURITY_GROUP:-false}
 NO_CNI=${NO_CNI:-false}
 CONFIGURE_CLUSTER_AUTOSCALER=${CONFIGURE_CLUSTER_AUTOSCALER:-false}
+CLUSTET_PREFIX=$(head -n 1 "${SHARED_DIR}/cluster-prefix")
```

Comment:
`CLUSTER_PREFIX` ? :)

Reply:
@radtriste This variable is used by `cluster_name`, `account-roles`, and `operator-roles` in this script. I do not think of the right name for these usages, but I inherit from the step directly. Any suggestion?

Reply:
Updated.